.github/workflows/operator-integration-test.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           kubectl wait --for=condition=Ready pod --all -n default
           kubectl get pods -A 
           kubectl describe pods -n default
-          go run integration-tests/manifests/cmd/validate_instrumentation_vars.go default integration-tests/java/default_instrumentation_java_env_variables.json
+          go run integration-tests/manifests/cmd/validate_instrumentation_vars.go default integration-tests/java/default_instrumentation_java_env_variables.json app_signals
```

Comment:
If your passing app_signals as an argument evertime, what is the point of the argument? Is there a case for not_app_signals? And if so, do you change the agent configuration to reflect that on the env vars?

Reply:
Yeah, I had to add it so it wouldn't cause a missing argument failure. The one time it is used is when we're checking for `no_app_signals`. Then it runs the check to make sure application signals environmental variables aren't present. However, I can adjust the logic to check the agent configuration itself.